Kconfig.nrf

```diff
@@ -25,6 +25,11 @@ config MBEDTLS_LIBRARY_NRF_SECURITY
 
 endchoice
 
+config TEST_RANDOM_GENERATOR
+	bool "Non-random number generator"
```

Comment:
Please remove this prompt @Vge0rge

Reply:
Conclusion. The build warning produced is not breaking the build. So we will keep the build warning instead of doing this specific override. Otherwise this would allow the configuration of a valid entropy driver and the test entropy driver to be selected at the same time.